src/renderers/dom/shared/DOMProperty.js

```diff
@@ -206,22 +206,12 @@ var DOMProperty = {
       return true;
     }
 
-    var lowerCased = name.toLowerCase();
-
-    var propertyInfo = DOMProperty.properties[name];
-
     switch (typeof value) {
       case 'boolean':
-        if (propertyInfo) {
-          return true;
-        }
-        var prefix = lowerCased.slice(0, 5);
-        return prefix === 'data-' || prefix === 'aria-';
```

Comment:
Just for clarity: we used to only allow booleans for non-flagged attributes on `data` and `aria` attributes. Now all attributes can accept booleans, cast to strings.